.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,92 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.database }}, ${{ matrix.phpcr-transport }})"
+        runs-on: ubuntu-18.04
+
+        env:
+            DATABASE_URL: mysql://root:root@127.0.0.1/sulu_redirect_test?serverVersion=5.7
+            DATABASE_CHARSET: utf8mb4
+            DATABASE_COLLATE: utf8mb4_unicode_ci
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.2'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      composer-flags: '--prefer-dist --no-interaction --prefer-lowest --prefer-stable'
+                      tools: 'composer:v1'
+                      lint: false
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: disabled
+                    - php-version: '7.4'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      composer-flags: '--prefer-dist --no-interaction'
+                      extensions: 'imagick'
```

Comment:
This variable isn't used anywhere. I would also add it to the `php7.2` job with `mysql, gd` and change this one to `mysql, imagick`

Reply:
And it should be called `php-extensions` I guess